README.rst

```diff
@@ -223,103 +219,99 @@ as an example.
    # Two KMS CMKs
    --master-keys provider=aws-kms key=$KEY_ARN_1 key=$KEY_ARN_2
 
-   # KMS Alias by name in default region
+   # KMS alias name in default AWS Region
    --master-keys provider=aws-kms key=$ALIAS_NAME
 
-   # KMS Alias by name in two specific regions
+   # KMS alias name in two AWS Regions
    --master-keys provider=aws-kms key=$ALIAS_NAME region=us-west-2
    --master-keys provider=aws-kms key=$ALIAS_NAME region=eu-central-1
 
 AWS KMS
 ```````
-If you want to use the ``aws-kms`` master key provider, you can either specify that
-as the provider or simply not specify a provider and allow the default value to be used.
+To use the ``aws-kms`` master key provider, you can either specify ``aws-kms`` in the provider attribute or omit the provider attribute.
 
-There are some configuration options which are unique to the ``aws-kms`` master key provider:
+You can use the following parameter attributes only with the ``aws-kms`` master key provider:
 
-* **profile** : Providing this configuration value will use the specified `named profile`_
-  credentials.
-* **region** : This allows you to specify the target region.
+* **region**  : Use the specified the target region.
+* **profile** : Use the credentials and region in the specified `named profile`_.
 
-The logic for determining which region to use is shown in the pseudocode below:
+The following logic determines which AWS Region to use:
 
 .. code-block:: python
 
    if key ID is an ARN:
-      use region identified in ARN
+      use the AWS Region in the ARN
    else:
-      if region is specified:
-         use region
-      else if profile is specified and profile has a defined region:
-         use region defined in profile
+      if an AWS Region is specified:
+         use it
+      else if profile is specified and profile includes an AWS Region:
+         use AWS Region in the profile
       else:
-         use system default region
+         use system default AWS Region
 
 Advanced Configuration
 ``````````````````````
-If you want to use a different master key provider, that provider must register a
+To use a different master key provider, that provider must register a
 `setuptools entry point`_. You can find an example of registering this entry point in the
 ``setup.py`` for this package.
 
-When a provider name is specifed in a call to ``aws-encryption-cli``, the appropriate entry
-point for that name is used.
+When you specify a provider name, the ``aws-encryption-cli`` uses the entry point for that name.
 
 Handling Multiple Entry Points
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-If multiple entry points are registered for a given name, you will need to specify the package
-that registered the entry point you want to use.
+If you specify multiple entry points are registered for a given name, you need to specify the package that registered the entry point you want to use.
```

Comment:
s/If you specify multiple entry points are registered/If multiple entry points are registered/

This is specifically identifying the scenario where multiple packages that you happen to have installed have registered entry points with the same name.